release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,262 @@ toc::[]
+
+* Permissions for permission sets will be grouped for simplification. The following list describes the new permissions and indicates the deprecated permissions that will be removed in a future release:
+** The `Access` permission will replace the following permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, and `User`.
+** The `DeploymentExtension` permission will replace the following permissions: `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, and `Risk`.
+** The `Integration` permission will deprecate the following permissions: `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, and `SignatureIntegration`.
+** The `Image` permission will replace the permission `ImageComponent`.
+* Previously, groups were retrieved by using the field `props` (`props.authProviderId`, `props.key`, `props.value`). This field will be replaced by the new `props.id` field. Use the `props.id` field to retrieve groups in the Application Programming Interface (API). Note the following:
+** Retrieval by using the `props` fields will be removed in version a future release.
+** Until removal, retrieval by using the `props` field will work if the result is unambiguous (no more than one group is found with the `props` field).
+* `/v1/cves/suppress` and `/v1/cves/unsuppress` have been deprecated and will be removed in a future release. After these are removed:
+** Use `/v1/imagecves/suppress` and `/v1/imagecves/unsuppress` to snooze and unsnooze image vulnerabilities.
+** Use `/v1/nodecves/suppress` and `/v1/nodecves/unsuppress` to snooze and unsnooze node and host vulnerabilities.
+** Use `/v1/clustercves/suppress` and `/v1/clustercves/unsuppress` to snooze and unsnooze platform (Kubernetes, Istio, and {ocp}) vulnerabilities.
+* Permission `ClusterCVE` is deprecated and will be superseded by the existing permission `Cluster`. 
+* `Label` and `Annotation` search options are deprecated and will be removed in the {product-title-short} 3.73 release. They will be replaced by the search options listed in the following table.
+
+.Search options
+[cols="1,1,1",options="header"]
+|====
+|Resource |Deprecated search option | New search option 
+
+| Resource
+| Deprecated Search Option
+| New Search Option
+
+| Node
+| Label
+| Node Label
+
+| Node
+| Annotation
+| Node Annotation
+
+| Namespace
+| Label
+| Namespace Label
+
+| Deployment
+| Label
+| Deployment Label
+
+| ServiceAccount
+| Label
+| Service Account Label
+
+| ServiceAccount
+| Annotation
+| Service Account Annotation
+
+| K8sRole
+| Label
+| Role Binding Label
+
+| K8sRoleAnnotation
+| Annotation
+| Role Binding Annotation
+
+|====
+
+
+[id="upcoming-api-changes"]
+=== Notice of upcoming API changes
+
+* CSV export: 
+** Beginning in the {product-title-short} 3.73 release, the CSV export `API /api/vm/export/csv` will require the `CVE Type` filter as part of the input query parameter. Requests that do not have the filter will time out and give an error. Examples:
```

Comment:
There won't be a timeout, just an error. So this can be updated to just say the requests will error out.